build.gradle

```diff
@@ -44,10 +59,6 @@ group = project.projectDir.name.toLowerCase()
 def modId = project.projectDir.name
 archivesBaseName = modId + "-mc" + minecraft_version
 
-sourceSets.main.java.srcDirs += 'java'
-sourceSets.main.java.srcDirs += 'apis'
-sourceSets.main.resources.srcDirs += 'resources'
```

Comment:
I know it's weird but I'd like to keep this file structure instead of `src/main/java`/`src/main/resources`. The API files can be in `java` and the apiJar will still work (that's [what I did for AppleCore](https://github.com/squeek502/AppleCore/blob/1.12.2/build.gradle#L58-L62)).